### ISSUE TITLE
Add a bit of movement on key up

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#218]
In #216 @jobara said 

> As a side note, I did notice a side effect of the way that the movement is calculated on keypress. If i tap rapidly, the balloon doesn't move at all. So it feels a bit broken. I think you could probably make it travel a minimum distance on keyup, this may also fix the JAWS issue.

Great idea @jobara we should give it a try.

**Comment by jessegreenberg** [comment]
I added a small position change on key up.   This *does* get the arrow keys to work with JAWS, but it doesn't work very well yet.

It feels like JAWS has a hard time processing the arrow key events.  If I press the left arrow key for 1 second, the balloon will continue to move to the left in a jagged way for ~4 seconds.  However, if I move the balloon with the WASD keys, it will only move while the key is down and moves with fluid motion.

This seems like a performance problem, and probably something on our end because other key event examples like https://jsfiddle.net/jk757xcz/5/show do not have this issue.

We could continue to investigate, but I would rather wait until we hear more from phetsims/a11y-research#9 before committing.

**Comment by jessegreenberg** [comment]
> This seems like a performance problem,

This is not a performance problem, it is the way that JAWS interprets the arrow keys under ```role=application```, see https://github.com/phetsims/a11y-research/issues/9.

In that issue, VFO suggested that this would be fixed in an improvement update.  There isn't anything else to do in this issue, closing.
